common/reviews/api/heft.api.md

```diff
@@ -38,6 +53,27 @@ export { CommandLineStringListParameter }
 
 export { CommandLineStringParameter }
 
+declare namespace ConfigurationFile {
```

Comment:
Yeah admittedly this is a bit odd, but I get why you're doing it. I'm still thinking re-exporting may be the best option. The alternate option would be to add a dev dep to projects that use this feature, which is IMO worse

Reply:
Unless bringing the world with Heft is the goal... which in that case, sure, this will work